lisa/core/node.py

```diff
@@ -108,18 +123,21 @@ def getToolPath(self, tool: Optional[Tool] = None) -> pathlib.Path:
         return tool_path
 
     def getTool(self, tool_type: Type[T]) -> T:
-        tool = cast(T, self.tools.get(tool_type))  # type: ignore
+        if tool_type is CustomScript:
```

Comment:
One of those implications I mentioned...

Reply:
I will move the CustomScript back to same tool flow, though it may be confusing in getTool itself. But it can manage dependencies better.